presto-session-property-managers/src/main/java/io/prestosql/plugin/session/SessionMatchSpec.java

```diff
@@ -14,6 +14,7 @@
 package io.prestosql.plugin.session;
```

Comment:
This change requires some documentation changes.

Reply:
Ok, lets move ahead with this format